src/EnergyPlus/CostEstimateManager.cc

```diff
@@ -683,6 +694,85 @@ namespace CostEstimateManager {
                         }
                     }
 
+                } else if (SELECT_CASE_var == "COIL:COOLING:DX") {
+
+                    auto &parentObjName = state.dataCostEstimateManager->CostLineItem(Item).ParentObjName;
+                    bool coilFound = false;
+                    thisCoil = 0;
+                    //  check for wildcard * in object name..
+                    if (parentObjName == "*") { // wildcard, apply to all such components
+                        WildcardObjNames = true;
+                    } else if (!state.dataCostEstimateManager->CostLineItem(Item).ParentObjName.empty()) {
+                        auto &v = state.dataCoilCooingDX->coilCoolingDXs;
+                        auto isInCoils = [&parentObjName](const CoilCoolingDX &coil) { return coil.name == parentObjName; };
```

Comment:
isInCoils sounds like a bool but is a coil name string. Just a little confusing.

Reply:
```isInCoils``` is a lambda that returns the ```return coil.name == parentObjName``` bool.